package.json

```diff
@@ -18,6 +18,7 @@
         "dotenv": "^16.1.3"
     },
     "devDependencies": {
-        "concurrently": "^8.1.0"
+        "concurrently": "^8.1.0",
+        "prisma": "^4.15.0"
```

Comment:
Wondering why Prisma is installed here and in `/db`?

Reply:
Fixed